### PR TITLE
Add Group Report Change

### PR DESCRIPTION
**Pull request by airicaruss**

### CODE REVIEW

**Review by Danielabom — CHANGES_REQUESTED** [review]
Thank you for working on this PR @airicaruss . I have a couple of questions. The api.md shows that you are adding properties, but they appear as methods in the changelog and they are not listed in the resource file. 
On the other hand, the methods are not listed in the resource file or the changelog.

**Comment by airicaruss** [comment]
> Thank you for working on this PR @airicaruss . I have a couple of questions. The api.md shows that you are adding properties, but they appear as methods in the changelog and they are not listed in the resource file. On the other hand, the methods are not listed in the resource file or the changelog.

@Danielabom updated the change to 'Property'. Please let me know if further change required

File: changelog/Microsoft.O365Reporting.json [hunk 1]
```diff
@@ -1917,6 +1917,32 @@
...
 		"SubArea": "Microsoft 365 usage reports"
+	},
+	{
+		"ChangeList": [
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "getOffice365GroupsActivityCounts",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessages**, **teamsMeetingsOrganized** property to the **getOffice365GroupsActivityCounts** resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1928]: One property per entry and the correct format in the description is: "Added the **property name** property to the [entity](https://learn.microsoft.com/en-us/graph/api/resources/entity?view=graph-rest-beta) resource.",

File: changelog/Microsoft.O365Reporting.json [hunk 2]
```diff
@@ -1917,6 +1917,32 @@
...
+			  "Target": "getOffice365GroupsActivityCounts"
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "getOffice365GroupsActivityDetail",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessagesCount**, **teamsMeetingsOrganizedCount** property to the **getOffice365GroupsActivityDetail** resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1936]: Please see my comments on the entry above.

File: changelog/Microsoft.O365Reporting.json [hunk 3]
```diff
@@ -1917,6 +1917,32 @@
...
+			  "Target": "getOffice365GroupsActivityDetail"
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1937]: Please see my comments on the entry above.

**Review by Danielabom — CHANGES_REQUESTED** [review]
@airicaruss, please read my comments. 
Based on the CSDL, these new properties belong to an entity called "office365GroupsActivityDetail", so please update the resource file.

File: changelog/Microsoft.O365Reporting.json [hunk 4]
```diff
@@ -1917,6 +1917,32 @@
 		"CreatedDateTime": "2022-10-12T00:00:00",
 		"WorkloadArea": "Reports",
 		"SubArea": "Microsoft 365 usage reports"
+	},
+	{
+		"ChangeList": [
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "getOffice365GroupsActivityCounts",
```
[the remark this answers is not in the document]
Comment (airicaruss) [reply changelog/Microsoft.O365Reporting.json]: Fixed

File: changelog/Microsoft.O365Reporting.json (hunk 1 above)
Comment (airicaruss) [reply changelog/Microsoft.O365Reporting.json]: Fixed

File: changelog/Microsoft.O365Reporting.json [hunk 5]
```diff
@@ -1917,6 +1917,32 @@
...
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessages**, **teamsMeetingsOrganized** property to the **getOffice365GroupsActivityCounts** resource.",
+			  "Target": "getOffice365GroupsActivityCounts"
```
[the remark this answers is not in the document]
Comment (airicaruss) [reply changelog/Microsoft.O365Reporting.json:1929]: The target resource is what I listed here. This is the CSDL. I found the doc said I can put the resource name here. Should I add some different value here? https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads?path=/Workloads/Microsoft.O365Reporting/override/schema-Prod-beta.csdl&version=GC4872263e3a2ab3f080eceeedc9de37e1cf6a993b

File: changelog/Microsoft.O365Reporting.json [hunk 6]
```diff
@@ -1917,6 +1917,32 @@
...
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "getOffice365GroupsActivityDetail",
```
[the remark this answers is not in the document]
Comment (airicaruss) [reply changelog/Microsoft.O365Reporting.json]: Fixed

File: changelog/Microsoft.O365Reporting.json (hunk 2 above)
Comment (airicaruss) [reply changelog/Microsoft.O365Reporting.json]: Fixed

**Comment by airicaruss** [comment]
@Danielabom please let me know if you have any other questions.

File: changelog/Microsoft.O365Reporting.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (Danielabom) [reply changelog/Microsoft.O365Reporting.json:1929]: Hi @airicaruss, your CSDL mentions that the entity resource name is **"office365GroupsActivityDetail**".
If you go to the topic [office365GroupsActivityDetail resource type - Microsoft Graph beta | Microsoft Learn](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta), you'll find a Properties table. New properties should be listed there.

File: changelog/Microsoft.O365Reporting.json [hunk 7]
```diff
@@ -1917,6 +1917,48 @@
...
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessages** property to the [getOffice365GroupsActivityCounts](https://learn.microsoft.com/en-us/graph/api/reportroot-getoffice365groupsactivitycounts?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityCounts"
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsMeetingsOrganized",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsMeetingsOrganized** property to the [getOffice365GroupsActivityCounts(https://learn.microsoft.com/en-us/graph/api/reportroot-getoffice365groupsactivitycounts?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1936]: On the other hand, **getOffice365GroupsActivityCounts** is not a resource; it is a function. Your description indicates something different.

File: changelog/Microsoft.O365Reporting.json [hunk 8]
```diff
@@ -1917,6 +1917,48 @@
...
+			  "Target": "getOffice365GroupsActivityCounts"
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsChannelMessagesCount",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessagesCount** property to the [getOffice365GroupsActivityDetail](https://learn.microsoft.com/en-us/graph/api/reportroot-getoffice365groupsactivitydetail?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityDetail"
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1945]: Global comment: Please update your target.

File: changelog/Microsoft.O365Reporting.json [hunk 9]
```diff
@@ -1917,6 +1917,48 @@
...
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsMeetingsOrganizedCount",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsMeetingsOrganizedCount** property to the [getOffice365GroupsActivityDetail(https://learn.microsoft.com/en-us/graph/api/reportroot-getoffice365groupsactivitydetail?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1952]: **getOffice365GroupsActivityDetail** is not a resource but a function. And this property belongs to an entity/resource called **office365GroupsActivityDetail**

**Review by Danielabom — CHANGES_REQUESTED** [review]
hi @airicaruss, please review my comments.

File: changelog/Microsoft.O365Reporting.json (hunk 7 above)
Comment (airicaruss) [reply changelog/Microsoft.O365Reporting.json]: Thank you for your comment. Fixed

File: changelog/Microsoft.O365Reporting.json (hunk 8 above)
Comment (airicaruss) [reply changelog/Microsoft.O365Reporting.json:1945]: Thank you for your comment. Fixed

File: changelog/Microsoft.O365Reporting.json (hunk 9 above)
Comment (airicaruss) [reply changelog/Microsoft.O365Reporting.json]: Thank you for your comment. Fixed

Comment (airicaruss) [reply changelog/Microsoft.O365Reporting.json]: @Danielabom updated the resource name. Could you please check?

**Review by Danielabom — CHANGES_REQUESTED** [review]
One more question, is there a particular reason why you are adding the properties to operations and not the resources?

**Comment by airicaruss** [comment]
> One more question, is there a particular reason why you are adding the properties to operations and not the resources?

@Danielabom no specific reason. Do you suggest to make changes on resource? What should I do in this scenario? Should I make change changes to this file as well? https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta

File: changelog/Microsoft.O365Reporting.json (hunk 8 above)
Comment (Lauragra) [reply changelog/Microsoft.O365Reporting.json:1945]: I think that per Dani's comment, the value of the Target field should be the resource name, not the function name. Also, we need to update the two resource topics to include the new properties.

**Comment by airicaruss** [comment]
> 

@Danielabom updated the resource file. Please take a look.

File: changelog/Microsoft.O365Reporting.json [hunk 10]
```diff
@@ -1917,6 +1917,48 @@
...
+			  "Description": "Added the **teamsChannelMessagesCount** property to the [office365GroupsActivityDetail](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityDetail"
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsMeetingsOrganizedCount",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsMeetingsOrganizedCount** property to the [office365GroupsActivityDetail(https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityDetail"
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1953]:
```suggestion
			  "Target": "Office365GroupsActivityDetail"
```

File: changelog/Microsoft.O365Reporting.json [hunk 11]
```diff
@@ -1917,6 +1917,48 @@
...
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessagesCount** property to the [office365GroupsActivityDetail](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityDetail"
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsMeetingsOrganizedCount",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsMeetingsOrganizedCount** property to the [office365GroupsActivityDetail(https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1952]:
```suggestion
			  "Description": "Added the **teamsMeetingsOrganizedCount** property to the [office365GroupsActivityDetail] (https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta) resource.",
```

File: changelog/Microsoft.O365Reporting.json [hunk 12]
```diff
@@ -1917,6 +1917,48 @@
...
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessagesCount** property to the [office365GroupsActivityDetail](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityDetail"
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsMeetingsOrganizedCount",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsMeetingsOrganizedCount** property to the [office365GroupsActivityDetail] (https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1952]:
```suggestion
			  "Description": "Added the **teamsMeetingsOrganizedCount** property to the [office365GroupsActivityDetail](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta) resource.",
```

File: changelog/Microsoft.O365Reporting.json [hunk 13]
```diff
@@ -1917,6 +1917,48 @@
...
+			  "Description": "Added the **teamsMeetingsOrganized** property to the [office365GroupsActivityCounts]https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitycounts?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityCounts"
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsChannelMessagesCount",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessagesCount** property to the [office365GroupsActivityDetail](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitydetail?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityDetail"
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1945]:
```suggestion
			  "Target": "Office365GroupsActivityDetail"
```

File: changelog/Microsoft.O365Reporting.json [hunk 14]
```diff
@@ -1917,6 +1917,48 @@
...
+			  "Description": "Added the **teamsChannelMessages** property to the [office365GroupsActivityCounts](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitycounts?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityCounts"
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsMeetingsOrganized",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsMeetingsOrganized** property to the [office365GroupsActivityCounts]https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitycounts?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityCounts"
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1937]:
```suggestion
			  "Target": "Office365GroupsActivityCounts"
```

File: changelog/Microsoft.O365Reporting.json [hunk 15]
```diff
@@ -1917,6 +1917,48 @@
...
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessages** property to the [office365GroupsActivityCounts](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitycounts?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityCounts"
+			},
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsMeetingsOrganized",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsMeetingsOrganized** property to the [office365GroupsActivityCounts]https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitycounts?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1936]:
```suggestion
			  "Description": "Added the **teamsMeetingsOrganized** property to the [office365GroupsActivityCounts](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitycounts?view=graph-rest-beta) resource.",
```

File: changelog/Microsoft.O365Reporting.json [hunk 16]
```diff
@@ -1917,6 +1917,48 @@
...
+	},
+	{
+		"ChangeList": [
+			{
+			  "Id": "a02cfa95-915d-411a-8acb-557c8d8aa659",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "teamsChannelMessages",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **teamsChannelMessages** property to the [office365GroupsActivityCounts](https://learn.microsoft.com/en-us/graph/api/resources/office365groupsactivitycounts?view=graph-rest-beta) resource.",
+			  "Target": "getOffice365GroupsActivityCounts"
```
Comment (Danielabom) [inline changelog/Microsoft.O365Reporting.json:1929]:
```suggestion
			  "Target": "Office365GroupsActivityCounts"
```

**Review by Danielabom — APPROVED** [review]
Applying changes and approving PR.

**Pull request merged**